rest/perf_routes.go

```diff
@@ -358,7 +358,70 @@ func (h *perfSignalProcessingRecalculateHandler) Run(ctx context.Context) gimlet
 		grip.Error(message.WrapError(err, message.Fields{
 			"request": gimlet.GetRequestID(ctx),
 			"method":  "POST",
-			"route":   "/perf/signal_processing/recalculate",
+			"route":   "/perf/change_points",
+		}))
+		return gimlet.MakeJSONErrorResponder(err)
+	}
+	return gimlet.NewJSONResponse(struct{}{})
+}
+
+///////////////////////////////////////////////////////////////////////////////
+//
+// POST /perf/change_points/triage/mark
+
+type changePointStub struct {
+	PerfResultID string `bson:"perf_result_id" json:"perf_result_id" yaml:"perf_result_id"`
+	Measurement  string `bson:"measurement" json:"measurement" yaml:"measurement"`
+}
+
+type changePointMarkRequest struct {
+	Status       string            `bson:"status" json:"status" yaml:"status"`
+	ChangePoints []changePointStub `bson:"change_points" json:"change_points" yaml:"change_points"`
```

Comment:
why stub? can we use an existing type from elsewhere?

Reply:
I don't think anything has this interface? It's meant to only be used to decode client request.